config.ml

```diff
@@ -123,9 +123,9 @@ let packages = [
 (* Network stack *)
 
 let stack =
-  match get_mode () with
-  | `Xen -> generic_stackv4 default_console tap0
-  | `Unix | `MacOSX -> socket_stackv4 default_console [Ipaddr.V4.any]
+  if_impl Key.is_xen
```

Comment:
more reliable: `is_unix` (since `is_xen` should be deprecated due to solo5 being around as well)